src/app/components/LoanTokenGraphs/index.tsx

```diff
@@ -3,37 +3,23 @@
  * LoanTokenGraphs
  *
  */
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import ParentSize from '@visx/responsive/lib/components/ParentSize';
-import { Bar } from '@visx/shape';
-import { Group } from '@visx/group';
-import { curveNatural } from '@visx/curve';
-import { scaleBand, scaleLinear } from '@visx/scale';
-import LinePath from '@visx/shape/lib/shapes/LinePath';
-import { Text } from '@visx/text/lib/index';
-import useTooltipInPortal from '@visx/tooltip/lib/hooks/useTooltipInPortal';
-import useTooltip from '@visx/tooltip/lib/hooks/useTooltip';
-// import styled from 'styled-components/macro';
```

Comment:
i think `@vix/` packages can be removed from package.json too because they were used only for lending charts.